worlds/civ_6/README.md

```diff
@@ -0,0 +1,53 @@
+# Civlization 6 Archipelago

```

Comment:
This is basically a duplicate of your `en_Civilization VI.md`

Reply:
Ah yep, that was for the apworld. I forgot to remove it here 👍